docs/src/main/sphinx/security/overview.rst

```diff
@@ -124,6 +124,18 @@ In addition, Trino :doc:`provides an API </develop/system-access-control>` that
 allows you to create a custom access control method, or to extend an existing
 one.
 
+The access control can limit access to columns of a table: default for ``SELECT *``
```

Comment:
.. of a table. The default behavior of an unqualified access to all columns with  a ``SELECT *`` statement is to deny access to all columns. 

You can change the behavior to silently hide inaccessible columns with the global property ``hide-inaccessible-columns`` in set in :ref:`config_properties`: 

.. code-block:: properties

    hide-inaccessible-columns=true